ghcup-vanilla-0.0.7.yaml

```diff
@@ -4172,6 +4172,76 @@ ghcupDownloads:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-fedora33.tar.xz
+              dlHash: 1e59dc1e1a1b33085a1789b8ddafb55026211454efe0b4e814c956ef86fe3ea5
+          Linux_Ubuntu:
+            '( >= 16 && < 19 )': &cabal-31020-64-ubuntu18
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-ubuntu18_04.tar.xz
+              dlHash: 7b0bb22c263ae0b43459de1995bf465560d412c12d47af011b2ac27b5d30c7aa
+            unknown_versioning: &cabal-31020-64-ubuntu20
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-ubuntu20_04.tar.xz
+              dlHash: c2a8048caa3dbfe021d0212804f7f2faad4df1154f1ff52bd2f3c68c1d445fe1
+          Linux_Mint:
+            '< 20': *cabal-31020-64-ubuntu18
+            '>= 20': *cabal-31020-64-ubuntu20
+          Darwin:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-darwin.tar.xz
+              dlHash: cd64f2a8f476d0f320945105303c982448ca1379ff54b8625b79fb982b551d90
+          Windows:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-windows.zip
+              dlHash: b09e335b2ebeafa1db5e1e5614e3e10fb37da230a36865d76646ab27b2f3f46b
+          FreeBSD:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-freebsd.tar.xz
+              dlHash: 6dcd9d38a9f2101a0a3c3b74cacb2e41b8f7226f181780c0f872f2f1206dee37
+        A_32:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-32
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-i386-linux-alpine3_12.tar.xz
+              dlHash: e219d2c6446c29e09644c4f064f4b87d486871ad6f6de16f0d2fcbd8626b5a5b
+          Linux_Alpine:
+            unknown_versioning: *cabal-31020-32
+          Linux_Debian:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb9.tar.xz
+              dlHash: 2b26d2cb67f1ba3561509fbccc30810ccc1c5032238fca00666e3dcd03e941a8
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-aarch64-darwin.tar.xz
+              dlHash: d2bd336d7397cf4b76f3bb0d80dea24ca0fa047903e39c8305b136e855269d7b
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-arm64
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-aarch64-linux-deb10.tar.xz
+              dlHash: 004ed4a7ca890fadee23f58f9cb606c066236a43e16b34be2532b177b231b06d
+          Linux_Debian:
```

Comment:
Can just drop this part... there's only one bindist for this platform.

Reply:
I stand corrected, we have two bindists here as well, please use them:

* https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-aarch64-linux-deb10.tar.xz
* https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-aarch64-linux-deb11.tar.xz